infrastructure-playbooks/rolling_update.yml

```diff
@@ -76,15 +79,11 @@
   become: True
 
   pre_tasks:
-    - include_vars: roles/ceph-common/defaults/main.yml
```

Comment:
I think this commit was before your change that introduced the `ansible.cfg` thing.